app/cdap/components/PipelineDetails/RunLevelInfo/CurrentRunIndex.tsx

```diff
@@ -14,27 +14,46 @@
  * the License.
  */
 
-import React from 'react';
+import React, { useEffect } from 'react';
 import { connect } from 'react-redux';
+import styled from 'styled-components'
 import IconSVG from 'components/shared/IconSVG';
 import { reverseArrayWithoutMutating, objectQuery } from 'services/helpers';
 import findIndex from 'lodash/findIndex';
 import {
   getAppVersion,
+  getRunsForVersion,
   init,
   setCurrentRunId,
 } from 'components/PipelineDetails/store/ActionCreator';
 import T from 'i18n-react';
 import { getCurrentNamespace } from 'services/NamespaceStore';
+import { getHydratorUrl } from 'services/UiUtils/UrlGenerator';
+import Popover from 'components/shared/Popover';
+import { GLOBALS } from 'services/global-constants';
 
 const PREFIX = 'features.PipelineDetails.RunLevel';
 
+const StyledDiv = styled.div`
```

Comment:
use contextual naming for these styled pieces instead of just styledElement - I know i've done this in the past too